### ISSUE TITLE
Piping works, but read from a file doesn't

### ISSUE THREAD

**Issue by dolmen** [maxlath/jsondepth#2]
jsondepth doesn't process input if stdin is a file.

``` sh
$ curl -s 'https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q1&format=json' > x.json
# This works:
$ cat x.json | jd
{ entities: [Object], success: 1 }
# This doesn't work (no output):
$ jd < x.json
$
```

**Comment by maxlath** [comment]
hey @dolmen, can you confirm that the patch works for you?

**Comment by dolmen** [comment]
Fixed on 1.3.3 :clap:
